### ISSUE TITLE
Error validating VPA

### ISSUE THREAD

**Issue by kamikazechaser** [grassrootseconomics/cic-ussd#109]
```bash
cic-ussd-1  | {"level":"DEBUG","time":"18-04-2024 12:39 PM","pid":18,"hostname":"XXX","name":"cic-ussd","msg":"State machine error occurred: \"Invalid phone number.\". STACKTRACE: MachineError: Invalid phone number.\n    at validatePhoneNumber (/app/dist/lib/ussd.js:222:15)\n    at validateUser (/app/dist/machines/utils.js:82:48)\n    at validateTargetUser (/app/dist/machines/utils.js:92:30)\n    at validateRecipient (/app/dist/machines/transfer.js:317:60)\n    at Interpreter._exec (/app/node_modules/xstate/lib/interpreter.js:269:63)\n    at handleAction (/app/node_modules/xstate/lib/actions.js:596:11)\n    at processBlock (/app/node_modules/xstate/lib/actions.js:627:24)\n    at Array.map (<anonymous>)\n    at Object.resolveActions (/app/node_modules/xstate/lib/actions.js:652:52)\n    at StateNode.resolveTransition (/app/node_modules/xstate/lib/StateNode.js:917:36)"}
```

**Comment by kamikazechaser** [comment]
[115,111,104,97,105,108,64,105,110,101,116,104,105] sohail@inethi
[115,111,104,97,105,108,64,105,110,101,116,104,105]	sohail@inethi

[115,111,104,97,105,108,64,105,110,101,116,104,105] sohail@inethi
[115,111,104,97,105,108,64,105,110,101,116,104,105] sohail@inethi

[107,111,114,105,121,111,119,194,161,100,97,100,97,97,98] 1*koriyowÂ¡dadaab
[107,111,114,105,121,111,119,161,100,97,100,97,97,98] 1*koriyow¡dadaab

[107,111,114,105,121,111,119,194,161,100,97,100,97,97,98]
[107,111,114,105,121,111,119,161,100,97,100,97,97,98]

[107,111,114,105,121,111,119,64,100,97,100,97,97,98]
[107,111,114,105,121,111,119,64,100,97,100,97,97,98]

**Comment by kamikazechaser** [comment]
Fixed in [bff84d4](https://github.com/grassrootseconomics/cic-ussd/commit/bff84d4b0caea58b92938fe19641e992c653ee35)
